.nuke/parameters.ci-chrome.json

```diff
@@ -3,9 +3,9 @@
   "Solution": "Remotion.sln",
   "Target": [ "CIBuild" ],
   "TestConfigurations": [ "Debug", "Release" ],
-  "TestExecutionRuntimes": [ "EnforcedLocalMachine" ],
+  "TestExecutionRuntimes": [ "EnforcedLocalMachine(Docker_Win_NET8_0)" ],
```

Comment:
@a-ctor when using the original value ("EnforcedLocalMachine", without the Docker-Argument), I get a warning in Rider that this isn't a supported value. When I update to include the Docker-Argument, I get a runtime error 
```
Target "CreateTestMatrix" has thrown an exception
 Target "CreateTestMatrix" has thrown an exception
    [ERR] Target CreateTestMatrix has thrown an exception
    System.InvalidOperationException: The supplied test dimension 'EnforcedLocalMachine(Docker_Win_NET8_0)' is not supported.
       at Remotion.BuildScript.Test.SupportedTestDimensions.ParseTestDimensionValuesOrDefault[T](String[] values)
       at Remotion.BuildScript.RemotionBuild.ConfigureEnabledTestDimensions(EnabledTestDimensionsBuilder enabledTestDimensions)
       at Build.ConfigureEnabledTestDimensions(EnabledTestDimensionsBuilder enabledTestDimensions) in C:\Build\28109e4c1456d59b\working\build\Build.cs:line 231
       at Remotion.BuildScript.Components.ITestMatrix.<get_CreateTestMatrix>b__10_1()
       at Nuke.Common.Execution.BuildExecutor.<>c.<Execute>b__4_2(Action x) in /_/source/Nuke.Build/Execution/BuildExecutor.cs:line 119
       at System.Collections.Generic.List`1.ForEach(Action`1 action)
       at Nuke.Common.Execution.BuildExecutor.Execute(NukeBuild build, ExecutableTarget target, IReadOnlyCollection`1 previouslyExecutedTargets, Boolean failureMode) in /_/source/Nuke.Build/Execution/BuildExecutor.cs:line 119
```
     
       
Any thoughts?

Reply:
It should stay as `EnforcedLocalMachine`, the "not supported value" is a bug in the schema generation.